scripts/docker/centos.7/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN yum -q -y install unzip libunwind gettext libcurl-devel openssl-devel zlib l
 
 # Install Build Prereqs
 # CMake 3.3.2 from GhettoForge; LLVM 3.6.2 built from source ourselves;
-RUN yum install -y http://mirror.symnds.com/distributions/gf/el/7/plus/x86_64/cmake-3.3.2-1.gf.el7.x86_64.rpm \
+RUN yum install -y http://mirror.ghettoforge.org/distributions/gf/el/7/plus/x86_64/cmake-3.3.2-1.gf.el7.x86_64.rpm \
         https://matell.blob.core.windows.net/rpms/clang-3.6.2-1.el7.centos.x86_64.rpm \
```

Comment:
@ellismg Can/should we mirror cmake-3.3.2-1.gf.el7.x86_64.rpm on matell.blob.core.windows.net too?